client/src/js/app.js

```diff
@@ -1,19 +1,63 @@
+import './dialog';
+
 document.addEventListener('DOMContentLoaded', () => {
   const adminToolbar = document.getElementById('admin-toolbar');
+  const adminToolbarToggle = document.querySelector('[data-toggle-admin-toolbar]');
+  const initialAdminToolbarActive = localStorage.getItem('ss-at-admin-toolbar-active');
+  const inactiveClass = 'ss-at-collapse';
 
-  if (!adminToolbar.classList.contains('admin-toolbar-collapsed')) {
-    document.body.classList.add('has-admin-toolbar');
+  if (initialAdminToolbarActive !== 'false') {
+    adminToolbar.querySelector('.admin-toolbar-inner').classList.remove(inactiveClass);
+  } else {
+    adminToolbarToggle.classList.toggle('ss-at-origin-center');
+    adminToolbarToggle.classList.toggle('ss-at-rotate-180');
   }
 
-  const adminToolbarToggle = document.getElementById('admin-toolbar-toggle');
-
   adminToolbarToggle.addEventListener('click', () => {
-    if (adminToolbar.classList.contains('admin-toolbar-collapsed')) {
-      adminToolbar.classList.remove('admin-toolbar-collapsed');
-      document.body.classList.add('has-admin-toolbar');
-    } else {
-      adminToolbar.classList.add('admin-toolbar-collapsed');
-      document.body.classList.remove('has-admin-toolbar');
-    }
+    const toolbar = adminToolbar.querySelector('.admin-toolbar-inner');
+    toolbar.classList.toggle(inactiveClass);
+
+    adminToolbarToggle.classList.toggle('ss-at-origin-center');
+    adminToolbarToggle.classList.toggle('ss-at-rotate-180');
+
+    localStorage.setItem('ss-at-admin-toolbar-active', !toolbar.classList.contains(inactiveClass));
+  });
+
+  const csrfToken = document.querySelector('.admin-toolbar-menu #SecurityID');
```

Comment:
Unneeded class lookup, can be straight upon the id.

```suggestion
  const csrfToken = document.getElementById('SecurityID');
```